xblockmufi/xblockmufi.py

```diff
@@ -72,27 +71,56 @@ def workbench_scenarios():
     """
     Main functions
     """
-    def student_view(self, context=None):
+    
+    def build_fragment(
+             self,
+             template,
+             context_dict,
+             initialize_js_func,
+             additional_css=[],
+             additional_js=[],
+     ):
+         #  pylint: disable=dangerous-default-value, too-many-arguments
```

Comment:
Here, we're silencing a warning that we shouldn't;
I think you can delete the `dangerous-default-value` after you fix the above dangerous defaults.